multiview_mapping_toolkit/cameras/cameras.py

```diff
@@ -731,6 +732,14 @@ def vis(
         if plotter is None:
             plotter = pv.Plotter()
             show = True
+        
+        # Determine pairwise distance between each camera and set frustum_scale to 1/120th of the maximum distance found
+        max_distance = 0
+        if frustum_scale is None and self.n_cameras() >= 2:
```

Comment:
If there is only one camera and no frustum scale, this breaks. This actually happens in the example concept figures notebook.